backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/pporder/PP_Order_StepDef.java

```diff
@@ -527,4 +538,28 @@ private static Map<String, ArrayList<String>> getPPOrderCandIdentifier2PPOrderId
 
 		return ppOrderCandidate2PPOrderIdsOrdered;
 	}
+
+	@And("load manufactured HU for PP_Order:")
+	public void loadManufacturedHUs(@NonNull final DataTable dataTable)
+	{
+		DataTableRow.toRows(dataTable)
+				.forEach(this::loadMAnufacturedHu);
+	}
+
+	private void loadMAnufacturedHu(@NonNull final DataTableRow tableRow)
+	{
+		final IQueryBuilder<I_PP_Order_Qty> builder = queryBL.createQueryBuilder(I_PP_Order_Qty.class)
+				.addEqualsFilter(I_PP_Order_Qty.COLUMNNAME_PP_Order_BOMLine_ID, null);
+		final I_PP_Order ppOrder = tableRow.getAsOptionalIdentifier(I_PP_Order_Qty.COLUMNNAME_PP_Order_ID)
+				.map(ppOrderTable::get)
+				.orElseThrow(() -> new AdempiereException("No PP_Order found for identifier " + tableRow.getAsString(I_PP_Order_Qty.COLUMNNAME_PP_Order_ID)));
+		builder.addEqualsFilter(I_PP_Order_Qty.COLUMNNAME_PP_Order_ID, ppOrder.getPP_Order_ID())
+				.addEqualsFilter(I_PP_Order_Qty.COLUMNNAME_M_Product_ID, ppOrder.getM_Product_ID());
+
+		final I_M_HU manufacturedHu = builder.andCollect(I_PP_Order_Qty.COLUMN_M_HU_ID)
```

Comment:
Can't ``I_PP_Order_Qty.COLUMNNAME_M_HU_ID`` be used as parameter?

Reply:
as discussed, both options could be used, none are deprecated. If it's an informal best practice, let's formalize it by deprecating one API and patch managing that PR to all active branches.